data_hub/services.py

```diff
@@ -118,12 +118,25 @@ def insert_fundamentals_data(self, df: pd.DataFrame, overwrite: bool = False):
             logging.error(f"Failed to insert fundamentals data: {e}")
             raise e
 
-    def get_macroeconomic_data(self, name: str) -> pd.DataFrame:
+    def get_macroeconomic_data(
+        self,
+        name: str,
+        from_date: str = None,
+        to_date: str = None,
+        year=None,
+        frequency=None,
+        month=None,
+        quarter=None,
+    ) -> pd.DataFrame:
         """
         Get historical macroeconomic data from BigQuery.
         """
+        return self.macroeconomic.get_data(
```

Comment:
```suggestion
        """

        return self.macroeconomic.get_data(
```
nit: blank line after docstring